src/deepsparse/loggers/metric_functions/registry.py

```diff
@@ -46,14 +47,15 @@ def some_function(image):
 
     def decorator(function):
         identifier_registry = None
-        task_registry = DATA_LOGGING_REGISTRY.get(group)
```

Comment:
Small refactor of the original `register`. Enables us to use a single decorator to map same identifier to a list of groups, to save lines of code.

so instead of 
```python
@register(group=object_detection", identifier="pipeline_outputs.scores")
@register(group="segmentation", identifier="pipeline_outputs.scores")
def func(...)
```

```python
@register(group=["object_detection", "segmentation"], identifier="pipeline_outputs.scores")
def func(...)
```